collect_app/build.gradle

```diff
@@ -137,6 +137,7 @@ allprojects {
     }
 }
 
+
```

Comment:
Before committing code, I usually do Cmd + K and go through the changes really quick to make sure there's no changes that I don't want gets committed. Sometimes I forget to that too ...

So revert this empty line.